pilvytis/order_status_tracker.go

```diff
@@ -139,6 +167,24 @@ func (t *StatusTracker) update() error {
 	return nil
 }
 
+func strEqual(s1, s2 *string) bool {
+	if s1 == nil {
```

Comment:
Maybe something like this would read better? https://play.golang.org/p/DuMx0-khcMX